Feat/dynamic step (#10958)
grafana/mimir: docs/sources/mimir/configure/configuration-parameters/index.md

# CLI flag: -query-frontend.shard-active-series-queries
[shard_active_series_queries: <boolean> | default = false]

> # (experimental) True to enable dynamic step calculation for range series queries.

Reviewer: ```suggestion
# (experimental) Set to true to enable dynamic step calculation for range series queries.
```

# CLI flag: -query-frontend.query-result-response-format
[query_result_response_format: <string> | default = "protobuf"]

> # (experimental) True to enable dynamic step calculation for range queries based

Reviewer: ```suggestion
# (experimental) Enables dynamic step calculation for range queries based
```

# on the query's complexity.
# CLI flag: -query-frontend.dynamic-step-enabled
[dynamic_step_enabled: <boolean> | default = false]

> # (advanced) Complexity threshold for dynamic step adjustment ( Only taken into
> # consideration if dynamic step is enabled ).

Reviewer: ```suggestion
# (advanced) Complexity threshold for dynamic step adjustment. This setting is only taken into
# consideration if you enable dynamic step.
```
